atom/browser/api/lib/auto-updater/auto-updater-win.js

```diff
@@ -2,70 +2,62 @@ const app = require('electron').app;
 const EventEmitter = require('events').EventEmitter;
 const url = require('url');
 const squirrelUpdate = require('./squirrel-update-win');
+const util = require('util');
 
-var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
-hasProp = {}.hasOwnProperty;
+function AutoUpdater() {
+  AutoUpdater.super_.call(this);
+}
 
-var AutoUpdater = (function(superClass) {
-  extend(AutoUpdater, superClass);
+util.inherits(AutoUpdater, EventEmitter);
 
-  function AutoUpdater() {
-    return AutoUpdater.__super__.constructor.apply(this, arguments);
-  }
-
-  AutoUpdater.prototype.quitAndInstall = function() {
-    squirrelUpdate.processStart();
-    return app.quit();
-  };
+AutoUpdater.prototype.quitAndInstall = function() {
+  squirrelUpdate.processStart();
+  return app.quit();
+};
 
-  AutoUpdater.prototype.setFeedURL = function(updateURL) {
-    return this.updateURL = updateURL;
-  };
+AutoUpdater.prototype.setFeedURL = function(updateURL) {
+  return this.updateURL = updateURL;
+};
 
-  AutoUpdater.prototype.checkForUpdates = function() {
-    if (!this.updateURL) {
-      return this.emitError('Update URL is not set');
-    }
-    if (!squirrelUpdate.supported()) {
-      return this.emitError('Can not find Squirrel');
-    }
-    this.emit('checking-for-update');
-    return squirrelUpdate.download(this.updateURL, (function(_this) {
-      return function(error, update) {
+AutoUpdater.prototype.checkForUpdates = function() {
+  if (!this.updateURL) {
+    return this.emitError('Update URL is not set');
+  }
+  if (!squirrelUpdate.supported()) {
+    return this.emitError('Can not find Squirrel');
+  }
+  this.emit('checking-for-update');
+  return squirrelUpdate.download(this.updateURL, (function(_this) {
+    return function(error, update) {
+      if (error != null) {
+        return _this.emitError(error);
+      }
+      if (update == null) {
+        return _this.emit('update-not-available');
+      }
+      _this.emit('update-available');
+      return squirrelUpdate.update(_this.updateURL, function(error) {
+        var date, releaseNotes, version;
         if (error != null) {
           return _this.emitError(error);
         }
-        if (update == null) {
-          return _this.emit('update-not-available');
-        }
-        _this.emit('update-available');
-        return squirrelUpdate.update(_this.updateURL, function(error) {
-          var date, releaseNotes, version;
-          if (error != null) {
-            return _this.emitError(error);
-          }
-          releaseNotes = update.releaseNotes, version = update.version;
+        releaseNotes = update.releaseNotes, version = update.version;
```

Comment:
Only the indenting was changed on this line so this is how that line was previously and so I think breaking it up is outside the scope of this pull request since the goal here to removing the CoffeeScript helpers.